key-vault.tf

```diff
@@ -1,29 +1,36 @@
 module "ccpay-vault" {
-  source = "git@github.com:hmcts/cnp-module-key-vault?ref=master"
+  source = "git@github.com:hmcts/cnp-module-key-vault?ref=azurermv2"
   name = "${var.product}-${var.env}"
-  product = "${var.product}"
-  env = "${var.env}"
-  tenant_id = "${var.tenant_id}"
-  object_id = "${var.jenkins_AAD_objectId}"
-  resource_group_name = "${azurerm_resource_group.rg.name}"
+  product = var.product
+  env = var.env
+  tenant_id = var.tenant_id
+  object_id = var.jenkins_AAD_objectId
+  resource_group_name = azurerm_resource_group.rg.name
   # group id of dcd_reform_dev_azure
   product_group_object_id = "56679aaa-b343-472a-bb46-58bbbfde9c3d"
-  common_tags         = "${var.common_tags}"
+  common_tags         = var.common_tags
   #aks migration
-  managed_identity_object_id = "${var.managed_identity_object_id}"
+  managed_identity_object_ids = ["${var.managed_identity_object_id}"]
+  create_managed_identity    = true
```

Comment:
This change is related to creating a shared MI. This can be removed, unless you are trying to create a shared MI along with this change.

Reply:
Removed create_managed_identity    = true as per review comments